src/hotspot/share/utilities/exceptions.cpp

```diff
@@ -123,6 +123,7 @@ bool Exceptions::special_exception(JavaThread* thread, const char* file, int lin
                         MAX_LEN, message ? message : "",
                         p2i(h_exception()), file, line, p2i(thread),
                         Universe::vm_exception()->print_value_string());
+    maybe_log_call_stack();
```

Comment:
This can't be coming from Java code, so don't log it here.  This is actually the compiler thread case that we should probably log the native stack trace for [JDK-8328639](https://bugs.openjdk.org/browse/JDK-8328639)